Aplib.Core.Tests/Intent/Tactics/TacticTests.cs

```diff
@@ -169,6 +169,21 @@ public void AnyOfTacticGetAction_WhenASubtacticIsEnabled_ReturnsActionOfEnabledS
         selectedAction.Should().Be(action1);
     }
 
+    [Fact]
+    public void AnyOfTactic_WithFailingGuard_ReturnsNoAction()
```

Comment:
This will be confusing with the new functionality of failing guards.
```suggestion
    public void AnyOfTactic_WithFalseGuard_ReturnsNoAction()
```